src/figuresClasses.ts

```diff
@@ -1,11 +1,109 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string = 'triangle';
+
+  color;
```

Comment:
The `color` property is declared but not initialized. Consider initializing it directly in the declaration or within the constructor.